src/chigame/api/tests/factories.py

```diff
@@ -90,12 +90,15 @@ class Meta:
 
     name = Faker("word")
     game = SubFactory(GameFactory)
-    start_date = Faker("date_this_year")
-    end_date = Faker("date_this_year")
-    max_players = 16
+    registration_start_date = Faker("date_this_year")
+    registration_end_date = Faker("date_this_year")
+    tournament_start_date = Faker("date_this_year")
+    tournament_end_date = Faker("date_this_year")
+    max_players = Faker("pyint", min_value=1, max_value=1000)
     description = Faker("text")
     rules = Faker("text")
     draw_rules = Faker("text")
+    num_winner = Faker("pyint", min_value=1, max_value=1000)
```

Comment:
When creating a new tournament, the website view field is called "Num winner," you should have a better name for it so it's a bit clearer what you're asking when creating the tournament. It's a good name for coding, but it should be changed for the user view on the website.

Reply:
I don't think these are changes handled by us. The only files we changed are src/chigame/urls.py, src/chigame/models.py, and src/chigame/views.py.